docs/data_management/data_responsibilities.md

```diff
@@ -0,0 +1,78 @@
+# Data Access Authorization and Responsibility
+
+Data access authorization and responsibility are critical aspect of managing and securing research data and resources. It ensures that only authorized individuals have access to specific data, maintaining security, compliance, and operational efficiency.
+
+## Shared Allocation
+
+A shared allocation is owned by a PI of a Lab or director of core facility. It is designed for sharing research data among lab members, and collaborators where permissions and access control are typically managed by the PI/director or designated administrators/manager.
+
+PIs/Core directors and their Lab members are responsible for overseeing and managing the allocations, including granting access to specific folders. However, Research Computing may provide support in certain cases. For example, if a folder becomes "locked" (i.e., no group members can change its permissions or access it), the the owner of the allocation or the folder should submit a request for us to fix the issue. In addition, if you need assistance configuring or reconfiguring permissions, we can provide support as a convenience. Simply send us a request via <support@listserv.uab.edu>.
```

Comment:
Check for "the the".

Let's aim to be more generic in our language here. "PIs/Core directors" could be "Shared storage owners", and "Lab members" could be "staff and students".

Change "Lab members" to "staff".